docs/docsite/rst/playbook_guide/playbooks_reuse_roles.rst

```diff
@@ -285,10 +285,19 @@ role ``meta/argument_specs.yml`` file. All fields are lower-case.
 
         * A short, one-line description of the entry point.
         * The ``short_description`` is displayed by ``ansible-doc -t role -l``.
+        * The short description should always be a string and never a list.
 
     :description:
 
         * A longer description that may contain multiple lines.
+        * This can be a single string, or a list of strings. In case this is a list of strings, every list
+           element is a new paragraph.
+
+    :version_added:
+    
+        * The version of the role when the entrypoint was added.
+        * This is a string, and not a float, for example, ``version_added: '2.1'``.
```

Comment:
While I would prefer if we don't accidentally recommend non-semver versioning by mentioning it works, this is a practical note.

Reply:
I'll happily change it then. :)